packages/plugin-compat/sources/extensions.ts

```diff
@@ -39,4 +39,17 @@ export const packageExtensions: Array<[string, any]> = [
       },
     },
   }],
+  // https://github.com/sindresorhus/got/pull/1125
+  [`got@*`, {
+    dependencies: {
+      [`@types/responselike`]: `^1.0.0`,
+      [`@types/keyv`]: `^3.1.1`,
+    },
+  }],
+  // https://github.com/szmarczak/cacheable-lookup/pull/12
```

Comment:
Doesn't seem like this is needed https://github.com/szmarczak/cacheable-lookup/pull/12#issuecomment-601638741